### ISSUE TITLE
Cashflow data is not accurate

### ISSUE THREAD

**Issue by mogcai** [ranaroussi/yfinance#816]
For example, 'Total Cash From Operating Activities' of '01398.HK' for '2020-12-31' retrieved from yfinance is -6.621400e+11. The figure shown on the yahoo finance website is 1,557,616,000.

```
data = yf.Ticker('1398.HK').cashflow
print(data.loc['Total Cash From Operating Activities'])
```

**Comment by euler007** [comment]
I can second this, the yahoo finance data and the data retrieved from the library are different

**Comment by ValueRaider** [comment]
Fixed in `yfinance` 0.2
